### ISSUE TITLE
All of sudden I started getting green backgrounds in the html files

### ISSUE THREAD

**Issue by dabeckham** [khzg/ChatGPT#1]
Prior to today, all of my saved chats are black text on white background in HTML file. Today, the responses started showing up as Black on Green for GPT-3.5 and Black on Black for GTP-4. And, as you can imagine, with GPT-4 being Blk on Blk, nothing is visible unless I highlight the text.

My last update was Feb 3, which is the date I first installed.

Any thoughts?

**Comment by khzg** [comment]
This was due to the openai update. i fixed it.
Update the browser plugin to version 3.

**Comment by dabeckham** [comment]
Thank you for the quick fix.  Would it be possible and easy to create a one button solution that would automatically iterate through all the conversations and save them to HTML files into a single directory?

**Comment by khzg** [comment]
nope, i think the program is more stable now and the user has more freedom.
